packages/mongo-nav/src/org-nav/OrgNav.tsx

```diff
@@ -115,46 +123,90 @@ function DropdownMenuIcon({ open }: { open: boolean }) {
   );
 }
 
-function MoreDropdownMenu({
-  children,
-  className,
+function GetHelpDropdownMenu({
   loading,
+  urls,
+  activeNav,
+  isTablet,
+  isMobile,
 }: {
-  children: React.ReactNode;
-  className?: string;
   loading: boolean;
+  urls: URLS['orgNav'];
+  activeNav?: ActiveNavElement;
+  isTablet?: boolean;
+  isMobile?: boolean;
 }) {
   const [isOpen, setIsOpen] = useState(false);
   const onElementClick = useOnElementClick();
 
   return (
     <OrgNavLink
+      isButton
+      data-testid="org-nav-dropdown-get-help"
+      isActive={activeNav === NavElement.OrgNavSupport}
       loading={loading}
-      className={className}
-      onClick={onElementClick(NavElement.OrgNavDropdownMoreMenu, () =>
+      className={rightLinkMargin}
+      onClick={onElementClick(NavElement.OrgNavDropdownGetHelp, () =>
         setIsOpen(open => !open),
       )}
-      isButton={true}
     >
-      More
+      {isMobile ? 'Help' : 'Get Help'}
       <DropdownMenuIcon open={isOpen} />
-      <Menu open={isOpen} setOpen={setIsOpen} usePortal={false}>
-        {children}
+      <Menu
+        open={isOpen}
+        setOpen={setIsOpen}
+        usePortal={false}
+        className={css`
+          width: 128px;
+        `}
+      >
+        <MenuItem
+          href={urls.support}
+          data-testid="org-nav-support-link"
+          active={activeNav === NavElement.OrgNavSupport}
+          size={isTablet ? 'large' : 'default'}
+          onClick={onElementClick(NavElement.OrgNavSupport)}
+        >
+          <div>Support</div>
+        </MenuItem>
+        <MenuItem
+          href={urls.docs}
+          data-testid="org-nav-docs-link"
+          size={isTablet ? 'large' : 'default'}
+          className={externalIconStyles}
+          onClick={onElementClick(NavElement.OrgNavDocs)}
+        >
+          <div
+            className={css`
+              display: flex;
+              align-items: center;
+              justify-content: space-between;
+            `}
+          >
+            Docs{' '}
```

Comment:
Is that empty string for spacing? We shouldn't do that if so.

Reply:
Also the new alignment probably makes this unnecessary anyways.

Reply:
👍